gdcdictionary/schemas/follow_up.yaml

```diff
@@ -1321,15 +1525,22 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  evidence_of_progression_type:
```

Comment:
@tlicht3 , should this be here?  I don't see it in the JIRA ticket?  Also if it is needed, it should have an entry in the terms file.

Reply:
@wwysoc2 ah - this was a discussion point in that long email from Megan. I think we need this because this is how the recurrence type is collected. I'll look into this and will update the terms file with this and the kidney staging one that needs added.